src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,14 @@
+server:
```

Comment:
any changes needed to application.yml?

Reply:
not sure will double check

Reply:
Looked at hmpps-prisoner-to-nomis-update project no config was required

Reply:
apparently uses helm vars and follows a convention to map them over to env vars